core/controllers/admin.py

```diff
@@ -129,6 +136,8 @@ def get(self):
             'continuous_computations_data': continuous_computations_data,
             'demo_explorations': demo_explorations,
             'demo_exploration_ids': demo_exploration_ids,
+            'demo_collections': demo_collections,
```

Comment:
I'm a bit confused by what you mean. Do you mean the demo collections should be sorted by their IDs? Since Python dicts don't have an inherent order, doesn't that mean this needs to be converted to a list of dicts?